src/lsm/table.zig

```diff
@@ -684,27 +680,59 @@ pub fn TableType(
             );
         }
 
+        pub inline fn index_data_addresses_used(index_block: BlockPtrConst) []const u64 {
+            const data_addresses = mem.bytesAsSlice(
+                u64,
+                index_block[index.data_addresses_offset..][0..index.data_addresses_size],
+            );
+            return data_addresses[0..index_data_blocks_used(index_block)];
```

Comment:
```suggestion
            const slice = mem.bytesAsSlice(
                u64,
                index_block[index.data_addresses_offset..][0..index.data_addresses_size],
            );
            return slice[0..index_data_blocks_used(index_block)];
```

To thin out the function body, and be consistent across all the `used()` methods.

For example, `index_data_keys_used()` currently has `keys` instead of `data_keys` whereas the others would have the `data_` prefix.

However, these `used()` functions also already repeat the subject three times, e.g. `keys` via `data_keys_offset`, `data_keys_size` and in the function name itself, so using something generic like `slice` is less visual overhead, puts the spotlight on the offset/size variables at play, and works across these functions consistently.